doc/src/hexdoc_hexcasting/_templates/hexcasting_render.js.jinja

```diff
@@ -319,19 +319,80 @@ let options = {
 };
 
 function load_render(name) {
-  if (selected == name) {
+  /*if (selected == name) {
     return;
-  }
-
-  selected = name;
+  }*/
 
-  if (name == "animated") {
+  if (name == "animated" && selected != name) {
     load_animated();
   } else {
     render_images(options[name], last_palette, name + "-settings");
   }
+
+  selected = name;
+}
+
+function checkEqual(ob1, ob2) {
```

Comment:
huh, none of the libraries we're using have a thing for this?

Reply:
hexdoc uses jquery and bootstrap:

https://github.com/hexdoc-dev/hexdoc/blob/b76ee52d09a1f5e829af0eb2b2f80629806a8569/src/hexdoc/_templates/index.html.jinja#L26-L38

Reply:
It doesn't seem like either includes a function like that.